basic/README.md

```diff
@@ -0,0 +1,18 @@
+# Basic Web Examples
+
```

Comment:
Can we get a proper description here of what the intent is of these examples? E.g. to provide basic building blocks or templates...